api/v1beta1/aerospikecluster_types.go

```diff
@@ -788,6 +788,10 @@ type AerospikePodStatus struct {
 	// initialized.
 	InitializedVolumes []string `json:"initializedVolumes,omitempty"`
 
+	// DirtyVolumes is the list of volume names that needs to be cleaned
```

Comment:
DirtyVolumes is the list of volume names that are removed from aerospike namespaces and will be cleaned during init if they are reused in any namespace